uint32 incubator_id
string name
float32 temperature 85.0
